### ISSUE TITLE
not an issue, just asking for help ! 

### ISSUE THREAD

**Issue by gadget78** [qeet/IMPORTJSONAPI#35]
how can i retrieve the value of ELS from this query...

=importJSONAPI("https://data.ripple.com/v2/accounts/rNJYAGgWCn6PcpNAaMnXWikn9eGbUkkMu5/balances", "$...balances[?(@property===('currency') && @.match===('ELS'))]", "@")

been trying so many different ways, nothing seems to work ! and im sure that it is possible !
 i couldn't find a link to any documents on using the query function ?
 many thanks for ANY pointers !

**Comment by qeet** [comment]
Try this:

`=importJSONAPI("https://data.ripple.com/v2/accounts/rNJYAGgWCn6PcpNAaMnXWikn9eGbUkkMu5/balances", "$...balances[[?(@property==='currency' && @.match('ELS'))]^", "@`

**Comment by gadget78** [comment]
YES !!! thankyou... that worked ..  i can see HOW and why now... easy when you shown hey ! 
Thank you so much, from the crypto world of XRP !
